{
  "active_lock_reason": null,
  "assignee": null,
  "assignees": [],
  "author_association": "CONTRIBUTOR",
  "body": "<!-- Please only use this template for submitting reports about failing tests in Kubernetes CI jobs -->\r\n\r\n**Which jobs are failing**:\r\ngce-cos-master-alpha-features\r\nconformance-GCE-master\r\nskew-cluster-latest-kubectl-stable1-gce\r\ngci-gce-ingress\r\ngce-cos-master-scalability-100\r\n\r\n**Which test(s) are failing**:\r\ntest start up\r\n\r\n**Since when has it been failing**:\r\n12th November 23:16 PST\r\n\r\n**Testgrid link**:\r\nhttps://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-alpha-features\r\n\r\n**Reason for failure**:\r\n\r\n```\r\nW1113 09:05:35.030] Failed to execute 'curl -s  http://127.0.0.1:4002/v2/members/$(curl -s  http://127.0.0.1:4002/v2/members -XGET | sed 's/{\\\"id/\\n/g' | grep bootstrap-e2e-master\\\" | cut -f 3 -d \\\") -XDELETE -L 2>/dev/null' on bootstrap-e2e-master despite 5 attempts\r\nW1113 09:05:35.030] Last attempt failed with: \r\nW1113 09:05:39.900] Updated [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/zones/us-west1-b/instances/bootstrap-e2e-master].\r\nW1113 09:06:44.042] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/zones/us-west1-b/instances/bootstrap-e2e-master].\r\nW1113 09:07:00.168] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-master-https].\r\nW1113 09:07:02.255] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-minion-all].\r\nW1113 09:07:06.404] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-master-etcd].\r\nW1113 09:07:12.867] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/regions/us-west1/addresses/bootstrap-e2e-master-ip].\r\nW1113 09:07:31.322] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-default-internal-master].\r\nW1113 09:07:32.079] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-default-internal-node].\r\nW1113 09:07:33.401] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-default-ssh].\r\nI1113 09:07:34.497] Deleting firewall rules remaining in network bootstrap-e2e: \r\nW1113 09:08:30.754] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/networks/bootstrap-e2e].\r\nW1113 09:08:31.142] W1113 09:08:31.140186    9616 loader.go:223] Config not found: /workspace/.kube/config\r\nI1113 09:08:31.407] Property \"clusters.k8s-boskos-gce-project-16_bootstrap-e2e\" unset.\r\nW1113 09:08:31.509] W1113 09:08:31.407002    9663 loader.go:223] Config not found: /workspace/.kube/config\r\nW1113 09:08:31.509] W1113 09:08:31.407293    9663 loader.go:223] Config not found: /workspace/.kube/config\r\nW1113 09:08:31.645] W1113 09:08:31.645324    9710 loader.go:223] Config not found: /workspace/.kube/config\r\nW1113 09:08:31.646] W1113 09:08:31.646206    9710 loader.go:223] Config not found: /workspace/.kube/config\r\nI1113 09:08:31.747] Property \"users.k8s-boskos-gce-project-16_bootstrap-e2e\" unset.\r\nW1113 09:08:31.856] W1113 09:08:31.856358    9758 loader.go:223] Config not found: /workspace/.kube/config\r\nW1113 09:08:31.857] W1113 09:08:31.857260    9758 loader.go:223] Config not found: /workspace/.kube/config\r\nI1113 09:08:31.959] Property \"users.k8s-boskos-gce-project-16_bootstrap-e2e-basic-auth\" unset.\r\nI1113 09:08:32.099] Property \"contexts.k8s-boskos-gce-project-16_bootstrap-e2e\" unset.\r\nI1113 09:08:32.102] Cleared config for k8s-boskos-gce-project-16_bootstrap-e2e from /workspace/.kube/config\r\nI1113 09:08:32.103] Done\r\nW1113 09:08:32.203] W1113 09:08:32.097713    9804 loader.go:223] Config not found: /workspace/.kube/config\r\nW1113 09:08:32.204] W1113 09:08:32.098096    9804 loader.go:223] Config not found: /workspace/.kube/config\r\nW1113 09:08:32.204] 2019/11/13 09:08:32 process.go:155: Step './hack/e2e-internal/e2e-down.sh' finished in 7m13.536227968s\r\nW1113 09:08:32.205] 2019/11/13 09:08:32 process.go:96: Saved XML output to /workspace/_artifacts/junit_runner.xml.\r\nW1113 09:08:37.490] 2019/11/13 09:08:37 main.go:319: Something went wrong: starting e2e cluster: error during ./hack/e2e-internal/e2e-up.sh: exit status 2\r\nW1113 09:08:37.490] Traceback (most recent call last):\r\nW1113 09:08:37.491]   File \"/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py\", line 778, in <module>\r\nW1113 09:08:37.493]     main(parse_args())\r\nW1113 09:08:37.493]   File \"/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py\", line 626, in main\r\nW1113 09:08:37.493]     mode.start(runner_args)\r\nW1113 09:08:37.494]   File \"/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py\", line 262, in start\r\nW1113 09:08:37.494]     check_env(env, self.command, *args)\r\nW1113 09:08:37.494]   File \"/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py\", line 111, in check_env\r\nW1113 09:08:37.494]     subprocess.check_call(cmd, env=env)\r\nW1113 09:08:37.494]   File \"/usr/lib/python2.7/subprocess.py\", line 190, in check_call\r\nW1113 09:08:37.495]     raise CalledProcessError(retcode, cmd)\r\nW1113 09:08:37.496] subprocess.CalledProcessError: Command '('kubetest', '--dump=/workspace/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--provider=gce', '--cluster=bootstrap-e2e', '--gcp-network=bootstrap-e2e', '--check-leaked-resources', '--extract=ci/latest', '--gcp-node-image=gci', '--gcp-zone=us-west1-b', '--runtime-config=api/all=true', '--test_args=--ginkgo.focus=\\\\[Feature:(Audit|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes|RunAsGroup|TTLAfterFinished|NodeLease|VolumeSnapshotDataSource|CSIInlineVolume)\\\\]|Networking --ginkgo.skip=Networking-Performance|IPv6|Feature:Volumes|csi-hostpath-v0 --minStartupPods=8', '--timeout=180m')' returned non-zero exit status 1\r\nE1113 09:08:37.505] Command failed\r\n```\r\n\r\n**Anything else we need to know**:\r\n/sig testing\r\n/milestone v1.17\r\n\r\n/priority critical-urgent\r\n\r\n/cc @droslean  @hasheddan @alejandrox1",
  "closed_at": "2019-11-14T03:33:51Z",
  "closed_by": {
    "avatar_url": "https://avatars0.githubusercontent.com/u/20407524?v=4",
    "events_url": "https://api.github.com/users/k8s-ci-robot/events{/privacy}",
    "followers_url": "https://api.github.com/users/k8s-ci-robot/followers",
    "following_url": "https://api.github.com/users/k8s-ci-robot/following{/other_user}",
    "gists_url": "https://api.github.com/users/k8s-ci-robot/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/k8s-ci-robot",
    "id": 20407524,
    "login": "k8s-ci-robot",
    "node_id": "MDQ6VXNlcjIwNDA3NTI0",
    "organizations_url": "https://api.github.com/users/k8s-ci-robot/orgs",
    "received_events_url": "https://api.github.com/users/k8s-ci-robot/received_events",
    "repos_url": "https://api.github.com/users/k8s-ci-robot/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/k8s-ci-robot/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/k8s-ci-robot/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/k8s-ci-robot"
  },
  "comments": 4,
  "comments_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/85196/comments",
  "created_at": "2019-11-13T10:03:23Z",
  "events_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/85196/events",
  "html_url": "https://github.com/kubernetes/kubernetes/issues/85196",
  "id": 522091534,
  "labels": [
    {
      "color": "e11d21",
      "default": false,
      "description": "Categorizes issue or PR as related to a consistently or frequently failing test.",
      "id": 496752468,
      "name": "kind/failing-test",
      "node_id": "MDU6TGFiZWw0OTY3NTI0Njg=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/kind/failing-test"
    },
    {
      "color": "e11d21",
      "default": false,
      "description": "Highest priority. Must be actively worked on as someone's top priority right now.",
      "id": 114528068,
      "name": "priority/critical-urgent",
      "node_id": "MDU6TGFiZWwxMTQ1MjgwNjg=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/priority/critical-urgent"
    },
    {
      "color": "d2b48c",
      "default": false,
      "description": "Categorizes an issue or PR as relevant to SIG Testing.",
      "id": 483069764,
      "name": "sig/testing",
      "node_id": "MDU6TGFiZWw0ODMwNjk3NjQ=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/sig/testing"
    }
  ],
  "labels_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/85196/labels{/name}",
  "locked": false,
  "milestone": {
    "closed_at": null,
    "closed_issues": 1674,
    "created_at": "2019-01-21T20:12:05Z",
    "creator": {
      "avatar_url": "https://avatars0.githubusercontent.com/u/980082?v=4",
      "events_url": "https://api.github.com/users/liggitt/events{/privacy}",
      "followers_url": "https://api.github.com/users/liggitt/followers",
      "following_url": "https://api.github.com/users/liggitt/following{/other_user}",
      "gists_url": "https://api.github.com/users/liggitt/gists{/gist_id}",
      "gravatar_id": "",
      "html_url": "https://github.com/liggitt",
      "id": 980082,
      "login": "liggitt",
      "node_id": "MDQ6VXNlcjk4MDA4Mg==",
      "organizations_url": "https://api.github.com/users/liggitt/orgs",
      "received_events_url": "https://api.github.com/users/liggitt/received_events",
      "repos_url": "https://api.github.com/users/liggitt/repos",
      "site_admin": false,
      "starred_url": "https://api.github.com/users/liggitt/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/liggitt/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/liggitt"
    },
    "description": null,
    "due_on": null,
    "html_url": "https://github.com/kubernetes/kubernetes/milestone/43",
    "id": 3990944,
    "labels_url": "https://api.github.com/repos/kubernetes/kubernetes/milestones/43/labels",
    "node_id": "MDk6TWlsZXN0b25lMzk5MDk0NA==",
    "number": 43,
    "open_issues": 17,
    "state": "open",
    "title": "v1.17",
    "updated_at": "2020-10-27T18:45:51Z",
    "url": "https://api.github.com/repos/kubernetes/kubernetes/milestones/43"
  },
  "node_id": "MDU6SXNzdWU1MjIwOTE1MzQ=",
  "number": 85196,
  "performed_via_github_app": null,
  "repository_url": "https://api.github.com/repos/kubernetes/kubernetes",
  "state": "closed",
  "title": "[Failing test] gce-cos-master-alpha-features (ci-kubernetes-e2e-gci-gce-alpha-features)",
  "updated_at": "2019-11-14T03:33:52Z",
  "url": "https://api.github.com/repos/kubernetes/kubernetes/issues/85196",
  "user": {
    "avatar_url": "https://avatars1.githubusercontent.com/u/264608?v=4",
    "events_url": "https://api.github.com/users/alenkacz/events{/privacy}",
    "followers_url": "https://api.github.com/users/alenkacz/followers",
    "following_url": "https://api.github.com/users/alenkacz/following{/other_user}",
    "gists_url": "https://api.github.com/users/alenkacz/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/alenkacz",
    "id": 264608,
    "login": "alenkacz",
    "node_id": "MDQ6VXNlcjI2NDYwOA==",
    "organizations_url": "https://api.github.com/users/alenkacz/orgs",
    "received_events_url": "https://api.github.com/users/alenkacz/received_events",
    "repos_url": "https://api.github.com/users/alenkacz/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/alenkacz/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/alenkacz/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/alenkacz"
  }
}